Gems/Terrain/Assets/Shaders/Terrain/TerrainClipmapDebugPass.azsl

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+ 
+#include <Atom/Features/SrgSemantics.azsli>
+
+#include <scenesrg.srgi>
+#include <viewsrg.srgi>
+
+ShaderResourceGroup PassSrg : SRG_PerPass_WithFallback
+{
+    Texture2DArray<float4> m_macroColorClipmaps;
```

Comment:
Would it be possible to just bind the TerrainSrg to this pass so you didn't need to bind all of the textures here?